.gitignore

```diff
@@ -6,3 +6,4 @@
 # Mac OS System
 .DS_Store*
 ._*
+/bin/
```

Comment:
Can you please move it up just above `/dist/`, since where you placed it is actually in the section for the "Mac OS System files", otherwise it will make the .gitignore file confusing